test/focusable-behavior.html

```diff
@@ -0,0 +1,86 @@
+<!doctype html>
```

Comment:
Do you need to add this file to `test/index.html`?

Reply:
Oh yes, thx.